packages/esm-patient-chart-app/src/clinical-views/encounter-tile/encounter-tile.component.tsx

```diff
@@ -65,7 +67,7 @@ const EncounterData: React.FC<{
       <span className={styles.tileTitle}>{t(column.header)}</span>
       {!(obsValue === '--' && summaryValue !== '--' && !isNil(summaryValue)) && (
         <span className={styles.tileValue}>
-          <p>{obsValue}</p>
+          <p>{withUnit(obsValue, units)}</p>
         </span>
       )}
 
```

Comment:
Should `summaryValue` on line 77 also be wrapped with `withUnit`?

Reply:
@denniskigen the gist is that this was done, but I asked to remove it because the summary value usually is used to show things like the `encounterDatetime`, but now that I've had some sleep (sorry, lol), I realize this was wrong. 
You're right, it should be wrapped in withUnits. @RajPrakash681 the `summaryObsValue` can have units, but different to the units of of the `obsValue`. So you need to wrap the `summaryObsValue` with `withUnits` but you need to get the unit of the summary obs. You can get its concept from `column.summaryConcept.primaryConcept`